src/disco/gui/fd_gui_printf.c

```diff
@@ -1002,8 +1002,15 @@ fd_gui_printf_slot( fd_gui_t * gui,
         else                                                 jsonp_ulong( gui, "failed_transactions", slot->failed_txn_cnt );
         if( FD_UNLIKELY( slot->compute_units==ULONG_MAX ) ) jsonp_null( gui, "compute_units" );
         else                                                jsonp_ulong( gui, "compute_units", slot->compute_units );
-        if( FD_UNLIKELY( slot->fees==ULONG_MAX ) ) jsonp_null( gui, "fees" );
-        else                                       jsonp_ulong( gui, "fees", slot->fees );
+        if( FD_UNLIKELY( slot->transaction_fee==ULONG_MAX || slot->priority_fee==ULONG_MAX ) ) {
+          jsonp_null( gui, "fees" );
```

Comment:
Don't need the "fees" field imo.

Reply:
Can you check? If it does error we can just have allen remove it and package that with this same commit.

Reply:
https://github.com/firedancer-io/firedancer-frontend/blob/b77f13657c72fc8408803a35413defb07502a9b5/src/api/entities.ts#L209

Maybe we just remove this line as part of this PR.

Reply:
Sure, if that fixes it feel free to push a commit to frontend